Dockerfile

```diff
@@ -103,6 +107,9 @@ RUN rm ${EMSCRIPTEN_TEMP}
 RUN cd ${EMSCRIPTEN_DIR} \
     && ./emsdk install ${EMSCRIPTEN_VERSION} \
     && ./emsdk activate ${EMSCRIPTEN_VERSION}
-ENV PATH "${EMSCRIPTEN_DIR}:${EMSCRIPTEN_DIR}/upstream/emscripten:${PATH}"
-ENV EMSDK "${EMSCRIPTEN_DIR}"
-ENV EM_CONFIG "/root/.emscripten"
+# Emscripten config file is automatically generated in `/root/`, so we need to allow access to it
```

Comment:
Is it generated in `/root` or in `$HOME`? I know they are the same here, but I think `$HOME` is more accurate, right?